app/assets/stylesheets/_style.css.scss

```diff
@@ -55,7 +58,7 @@ nav {
       color: $light-gray;
       display: inline-block;
       font-size: 1rem;
-      height: $_nav_height;
+      height: $nav_height;
```

Comment:
Name of variable `nav_height` should be written in all lowercase letters with hyphens instead of underscores